jaeger-core/src/test/java/io/jaegertracing/internal/ActiveSpanTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016, Uber Technologies, Inc
+ * Copyright (c) 2018, The Jaeger Authors.
```

Comment:
`Copyright (c) 2016-2017, Uber Technologies, Inc`?

Reply:
Nope, considering that active span did not exist in 2016 and most of 2017, and it was all implemented by people not from Uber.